### ISSUE TITLE
remote syslog

### ISSUE THREAD

**Issue by philtay** [pgbouncer/pgbouncer#99]
As of today PgBouncer can send logs only to a local syslog daemon.
This turns out to be a problem when PgBouncer is deployed in a microservices architecture, where the syslog daemon most likely resides on another host.
For instance, in Docker, you are forced to run two supervised processes in the same container (pgbouncer + syslog) or redirect the logs to stdout and use "docker logs".

The cleanest solution would be to allow the user to specify the address of a syslog server. It can be local (unix socket) or remote.
Basically, exactly what nginx already does: http://nginx.org/en/docs/syslog.html

**Comment by markokr** [comment]
Yeah, it could be a good feature.

Care to submit a patch? :)
